src/eval-frame/actions/editor-message-senders.js

```diff
@@ -0,0 +1,9 @@
+import messagePasserEval from "../../redux-to-port-message-passer";
+
+export function sendStatusResponseToEditor(status, evalId) {
+  messagePasserEval.postMessage("EVALUATION_RESPONSE", { status, evalId });
+}
+
+export function sendActionToEditor(action) {
+  messagePasserEval.postMessage("REDUX_ACTION", action);
+}
```

Comment:
this is just a little convenience fn that made it easy to migrate from `dispatch(actionCreatorFnc(...))` to `sendActionToEditor(actionCreatorFnc(...))`. may not keep this in the long run, but it made the migration flow tractable